packages/react-native-web/src/exports/StyleSheet/preprocess.js

```diff
@@ -182,7 +182,7 @@ export const preprocess = <T: {| [key: string]: any |}>(
       continue;
     }
 
-    if (prop === 'aspectRatio') {
+    if (prop === 'aspectRatio' && value !== undefined) {
```

Comment:
This check should probably be `!= null`

Reply:
The thing is aspectRatio's type is either string, number, or undefined. If null was used, then a type error should be shown I guess, but undefined is valid. That's why I put a check for undefined. If we dont check for undefined, then the app crashes.

Reply:
We can do (typeof aspectRatio === 'number' || typeof aspectRatio === 'string') if you want. This way we check it is what we want rather than checking the opposite